tool/generator/lib/src/projection/winrt/declarations/datetime.dart

```diff
@@ -0,0 +1,63 @@
+import '../../../../generator.dart';
+
+class DateTimeMethod extends WinRTMethodProjection {
```

Comment:
Whatever name convention we adopt above for COMObject, we should replicate for other methods.

Reply:
Done